governance.md

```diff
@@ -18,88 +36,180 @@ The committee MUST:
+
+Details about the schedule and logistics of the election will be announced in a timely manner by the election officers to eligible candidates and voters via the cdevents-dev@googlegroups.com mailing list.
+
+Example timeline:
+
+1. Before or during nominations, send an email to all eligible voters to notify them that they are eligible (allowing people time to reach out if they believe they are eligible but are not on our list)
+1. Prompt eligible voters to opt into CIVS at https://civs1.civs.us/cgi-bin/opt_in.pl so they will be able to vote
+1. 1 week for nominations (previously, starting on a Thursday until midnight UTC the next Wednesday)
+1. 1 week for the election itself (starting the following Thursday until midnight UTC the next Wednesday)
+
+### Election Officers
+
+For every election, the governance board wll choose three election officers, by the following criteria, so as to promote healthy rotation and diversity:
+
+- election officers must be eligible to vote
+- two election officers should have served before
+- one election officer should have never served before
+- each officer should come from a different company to maintain 40% maximal representation
+- election officers should not be currently running in the election
+
+The governing board can decide to make exceptions to the above requirements if needed (for example, if two people cannot be found who have served before and want to be officers).
+
+### Vacancies
+
+In the event of a resignation or other loss of an elected governance board member, the candidate with the next most votes from the previous election will be offered the seat. This process will continue until the seat is filled.
+
+In case this fails to fill the seat, a special election for that position will be held as soon as possible. Eligible voters from the most recent election will vote in the special election (ie: eligibility will not be redetermined at the time of the special election). A board member elected in a special election will 
+serve out the remainder of the term for the person they are replacing, regardless of the length of that remainder.
+
+### Changes to governing board
+
+When someone joins the governing board:
+
+- They should be added to [the list of current governing board members](#current-members)
+- They will be added as admins to [the CDEvents GitHub org](https://github.com/cdevents/) via the `governance` GitHub team
+- They will be added to these mailing lists as owners:
+  - [`cdevents-governance`](https://groups.google.com/g/cdevents-governance)
+  - [`cdevents-nominations`](https://groups.google.com/g/cdevents-nominations)
+  - [`cdevents-dev`](https://groups.google.com/g/cdevents-dev)
+  - [`cdevents-users`](https://groups.google.com/g/cdevents-users)
+  - [`cdevents-code-of-conduct`](https://groups.google.com/g/cdevents-code-of-conduct)
```

Comment:
Today we use [conduct@cd.foundation](mailto:conduct@cd.foundation) - so I will remove this - @fdegir do you think this is ok?

Reply:
Yep, it is better to have CDEvents specific medium for handling conduct issues.

Reply:
fixed